css-uc/styles.css

```diff
@@ -0,0 +1,339 @@
+/* General properties */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --white: #FFFFFF;
+  --blue-gradient: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%);
+  --black: #1C1C1C;
+  --lightblue-gradient: linear-gradient(46.77deg, #0F99B7 -0.79%, #69ECDC 131.52%);
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Poppins';
+  background-color: var(--white);
+  margin-inline: auto;
+  width: 80%;
+}
+
+/* Flex properties*/
+.flex {
+  display: flex;
+}
+
+.flex-col {
+  flex-direction: column;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.justify-around {
+  justify-content: space-around;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.items-center {
+  align-items: center;
+}
+
+.items-start {
+  align-items: flex-start;
+}
+
+/* Margins, paddings and gaps */
+.mt-1 {
+  margin-top: 10px;
+}
+
+.mt-7 {
+  margin-top: 70px;
+}
+
+.mt-6 {
+  margin-top: 60px;
+}
+
+.mt-14 {
+  margin-top: 140px;
+}
+
+.mt-22 {
+  margin-top: 220px;
+}
+
+.p-0 {
+  padding: 0px;
+}
+
+.p-btn1 {
+  padding: 5px 40px;
+}
+
+.p-btn2 {
+  padding: 9.5px 53px;
+}
+
+.p-btn3 {
+  padding: 8px 40px;
+}
+
+.gap-1 {
+  gap: 10px;
+}
+
+.gap-2 {
+  gap: 20px;
+
+}
+
+.gap-3 {
+  gap: 30px;
+}
+
+.gap-4 {
+  gap: 40px;
+}
+
+.gap-6 {
+  gap: 60px;
+}
+
+.gap-15 {
+  gap: 150px;
+}
+
+.gap-20 {
+  gap: 200px
+}
+
+/* Lists properties */
+.list-none {
+  list-style: none;
+}
+
+/* Text sizes, colors, align and letter spacing*/
+.text-2xl {
+  font-size: 6.0rem;
+}
+
+.text-xl {
+  font-size: 4.8rem;
+}
+
+.text-l {
+  font-size: 3.5rem;
+}
+
+.text-s {
+  font-size: 3.0rem;
+}
+
+.text-b {
+  font-size: 1.6rem;
+}
```

Comment:
Let's follow the naming conventions defined for font-sizes. The names should be `base` `sm` and `lg`